src/ui/components/SecondaryToolbox/SecondaryToolbox.css

```diff
@@ -17,7 +17,7 @@
   display: flex;
   flex-direction: row;
   align-items: center;
-  overflow: hidden;
+  overflow: auto;
```

Comment:
Horizontally scroll if we need to.

Reply:
Have you checked this on the other SecondaryPanels? Sometimes those things have an annoying amount of interdependency on the same CSS rules.

Reply:
I'll leave this comment here because it's good context but I just clicked through on the preview branch and didn't see anything obvious 👍